GeneratorInterface/Core/interface/PartonShowerCsHepMCFilter.h

```diff
@@ -0,0 +1,30 @@
+/**
+ ** Description: Filter gen particles based on pdg_id and status code
+ ** 
+ ** @author bortigno
+ ** @version 1.0 02.04.2015
+*/
+
+
+#ifndef __PARTONSHOWERCSHEPMCFILTER__
```

Comment:
@mastrolorenzo could you please use here and in the line below a more standard include guard name "PartonShowerCsHepMCFilter_h" name?